### PR TITLE
vale and mklint editorial updates - CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by grantnoble**
Line length <= 80 where possible (mklint).
Fixed the How to Contribute link (vale).
Misc capitalization and wording changes.

### CODE REVIEW

**Comment by NicolasMassart** [comment]
100 chars is really already enough, but if 80 is readable, that's ok.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
 # Contributing to the Hyperledger Besu Documentation
-## :tada: Thanks for taking the time to contribute! :tada:
 
+## Thanks for taking the time to contribute!
```
Comment (NicolasMassart) [inline CONTRIBUTING.md]: Maybe we should not make it a heading as the ! will break the linter.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -13,19 +14,16 @@ and feel free to propose changes to this document in a pull request.
...
 * [Documentation Style Guide](https://wiki.hyperledger.org/display/BESU/Besu+Documentation+Style+Guide)
 * [Pull Request Labels](https://wiki.hyperledger.org/display/BESU/Pull+Request+Labels)
-
 * [Security](SECURITY.md)
 
-
 ### Other important information
 
-* [Besu Roadmap](https://wiki.hyperledger.org/display/BESU/Roadmap)
+* [Besu roadmap](https://wiki.hyperledger.org/display/BESU/Roadmap)
```
Comment (NicolasMassart) [inline CONTRIBUTING.md]: Do other titles like "Code of Conduct", "Submitting a Documentation Enhancement Suggestion" , "Documentation Style Guide" and "Pull Request Labels" have to be sentence cased too?

File: CONTRIBUTING.md (hunk 1 above)
Comment (grantnoble) [reply CONTRIBUTING.md]: I'll remove the !

File: CONTRIBUTING.md (hunk 2 above)
Comment (grantnoble) [reply CONTRIBUTING.md]: I would have thought so if that is our style.

Comment (grantnoble) [reply CONTRIBUTING.md]: Updated, plus fixed up the long links.

**Pull request merged**